docs/install-a-package.md

```diff
@@ -0,0 +1,35 @@
+You can install extensions available on the database.dev registry using the dbdev CLI's `add` command. The dbdev client is itself an extension which you can install by following the instructions below.
+
+## Pre-requisites
+
+Before you can install a package, ensure you have the `pg_tle` extension installed in your database.
+
+!!! note
+
+    If your database is running on Supabase, `pg_tle` is already installed.
+
+!!! warning
+
+    Restoring a logical backup of a database with a TLE installed can fail. For this reason, dbdev should only be used with databases with physical backups enabled.
+
+```sql
+create extension if not exists pg_tle;
+```
+
+## Use
+
+Once the prerequisites are met, you can create a migration file to install a TLE available on database.dev by running the following dbdev command:
+
+```bash
+dbdev add -c <postgres_connection_string> -o <migration_folder_path> package -n <package_name>
```

Comment:
why is the postgres connection string needed for this?

I expected it'd produce a SQL script which could then be copied into the migrations directory vs do anything with a live instance

then they'd run the supabase migrations in the usual way to install it. That'd keep the tools decoupled

Reply:
The problem is with updates. If someone has installed an existing version and wants to update to a newer version of a package, the newly generated migration file has to only include calls to pg_tle API for newer versions/upgrade paths. The connection to Postgres is used to figure out which versions and upgrade paths already exist in the database and include only those in the migration file.

What we probably can do is to make the postgres connection optional. If it is not supplied, we assume that the migration file has to be generated for first time installation. If it is supplied then we handle the update case.